Python/Product/PythonTools/PythonTools/Environments/InterpreterView.cs

```diff
@@ -29,8 +29,9 @@ sealed class InterpreterView : DependencyObject {
 
         public static IEnumerable<InterpreterView> GetInterpreters(
             IServiceProvider serviceProvider,
-            PythonProjectNode project, 
-            bool onlyGlobalEnvironments = false
+            PythonProjectNode project,
+            bool onlyGlobalEnvironments = false,
```

Comment:
`onlyGlobalEnvironments` is a form of exclusion so it feels like it should be replaced by the new parameter.

Reply:
onlyGlobalEnvironments requires IInterpreterRegistryService which is not accepted by ExcludeInterpreter. I could modify ExcludeInterpreter, but it would be somewhat weird (IInterpreterRegistry only passed in case of ExcludeGlobalInterpreter) and I'd have to write additional tests for ExcludeGlobalInterpreter cases. I don't think the cost vs benefit is justified when everything works fine right now.